src/frontend/js/components/DashboardItem/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC, ReactNode } from 'react';
+
+interface DashboardItemProps {
+  title: string;
+  code: string;
+  imageUrl: string;
+  footer?: ReactNode;
+}
+
+export const DEMO_IMAGE_URL =
+  'https://dz03nossv7tsm.cloudfront.net/media/filer_public_thumbnails/filer_public/12/32/123267a1-bc28-49ca-bfca-679568c34043/logo_upsaclay-2020.jpg__200x113_q85_replace_alpha-%23FFFFFF_subsampling-2_upscale.jpg';
+
+export const DashboardItem: FC<DashboardItemProps> = (props) => {
+  return (
+    <div className="dashboard-item">
+      <div className="dashboard-item__block">
+        <header className="dashboard-item__block__head">
+          <img
+            data-testid="dashboard-item__block__head__thumbnail"
+            className="dashboard-item__block__head__thumbnail"
+            src={props.imageUrl}
+            alt=""
+          />
+          <div className="dashboard-item__block__head__captions">
+            <div className="dashboard-item__block__head__code">{props.code}</div>
+            <h5 className="dashboard-item__block__head__title">{props.title}</h5>
```

Comment:
hey, hope it's OK for me to make some accessibility related comments :innocent: 

I'd argue here it'd be better if the heading appeared before the code in the DOM. As people with screen reader can navigate by jumping from heading to heading, if an information related to some heading appears in the DOM before it, it can be missed (since a screen reader then reads the content in the order of the DOM).

So the idea here would be to swap the `__code` and `__title` lines and rearrange them visually via flexbox + css `order` prop, for example.

Reply:
Hey, of course it's ok ! I just made the change. I will learn lots of things :)